### ISSUE TITLE
The results can't be reproduced

### ISSUE THREAD

**Issue by yuzhou2333** [yuqianghan/editretro#7]
Dear author,when i set the --max_token is 2500,the generate can run correctly.

![image](https://github.com/user-attachments/assets/a4201d96-f0f5-4cbc-9b8b-b54b5db95c97)

The reproduced results are slightly worse than the results reported in the paper.All the code is run on a single 3090 GPU, with the commands executed in the following order:
bash ./scripts/0_pretrain.sh 0
bash ./scripts/1_finetune.sh 0
bash ./scripts/2_generate.sh 0

**Comment by Syforeal** [comment]
> Dear author,when i set the --max_token is 2500,the generate can run correctly.
> 
> ![image](https://private-user-images.githubusercontent.com/74141246/362503678-a4201d96-f0f5-4cbc-9b8b-b54b5db95c97.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjUwMDc3MTcsIm5iZiI6MTcyNTAwNzQxNywicGF0aCI6Ii83NDE0MTI0Ni8zNjI1MDM2NzgtYTQyMDFkOTYtZjBmNS00Y2JjLTliOGItYjU0YjVkYjk1Yzk3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MzAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODMwVDA4NDMzN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWI2OTU1ZGI1MmUxNTZiMjgyMDZhZTdkYjdiYzY2MDA0YWRhODc0NDJjYTQ5OTIyNDc2OGZkZmFmMDgyNWUyYjUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.jaj0TahmWaEZPU_WB0hOcpaYTzleB3EyGQTebAa0zfg)
> 
> The reproduced results are slightly worse than the results reported in the paper.All the code is run on a single 3090 GPU, with the commands executed in the following order: bash ./scripts/0_pretrain.sh 0 bash ./scripts/1_finetune.sh 0 bash ./scripts/2_generate.sh 0

Hello!May I ask about the parameter settings you used during training? I would also like to know how the loss reduction progressed when training with a GPU. Could you please share it for my reference?  The results I obtained with GPU training are not ideal as yours, as the loss stopped improving after decreasing to around 3.0*.Thank you so much!

**Comment by Syforeal** [comment]
Hello!May I ask about the parameter settings you used during training? I would also like to know how the loss reduction progressed when training with a GPU. Could you please share it for my reference? The results I obtained with GPU training are not ideal as yours, as the loss stopped improving after decreasing to around 3.0*.Thank you so much!

**Comment by yuqianghan** [comment]
Thank you for your interest in our work. We will soon update the pretraining instructions, including the data and checkpoints. I will let you know of any updates.

**Comment by yuqianghan** [comment]
We have updated the pretraining and finetune instructions. Welcome to try the new scripts.
